core/modules/layout/layout.theme.inc

```diff
@@ -46,8 +46,44 @@ function theme_layout_reorder_layouts($variables) {
  */
 function theme_layout_info($variables) {
   $layout = $variables['layout'];
+  $info = layout_get_layout_info($layout->layout_template);
```

Comment:
`layout_get_layout_info()` is deprecated, use `layout_get_layout_template_info()`.